content/articles/object-based-image-analysis-using-matlab/index.md

```diff
@@ -0,0 +1,241 @@
+
+Images contain objects with distinct regions. These regions have area, perimeter, shapes, and height features. Matlab allows for analysis of these properties using image analyzer functions or region props to obtain data from these images.
+
+Object-based image analysis is useful, especially in analyzing satellite maps, machine vision, and obtaining information based on object characteristics in an image.
+
+### Table of contents
+- [Prerequisites](#prerequisites)
+- [Edge detection](#edge-detection)
+- [Methods of edge detection](#methods-of-edge-detection)
+- [Object boundary detection](#object-boundary-detection)
+- [Boundary detection using morphological operations](#boundary-detection-using-morphological-operations)
+- [Obtaining boundary by erosion](#obtaining-boundary-by-erosion)
+- [Obtaining boundary by dilation](#obtaining-boundary-by-dilation)
+- [Highlighting text in an image](#highlighting-text-in-an-image)
+- [Labeling objects in images](#labeling-objects-in-images)
+- [Conclusion](#conclusion)
+
+### Prerequisites
+To follow along with this tutorial, you'll need:
+- [MATLAB](https://www.mathworks.com/products/get-matlab.html?s_tid=gn_getml) installed.
+- Basic understanding of [MATLAB](https://www.section.io/engineering-education/getting-started-with-matlab/) basics.
+
+
+### Edge detection 
+Edge detection is the identification of points within an image. These points are where the image has excellent contrast, and they are the defining points of the image. Edge detection work by detecting changes in brightness of the image pixels.
+
+Edge detection is useful in image segmentation and data extraction for comparison, objects separation, computer vision, and machine learning. Matlab supports a variety of functions that aids in edge detection.
+
+### Methods of edge detection
+There are several methods used in edge detection in images. These methods are used with the primary function `edge()`.  We will demonstrate most of these methods using a similar image and then compare the outcome to find a more suitable way to provide a more detailed output.
+
+All these methods only accept a grayscale image input; hence, converting the RGB image to a grayscale image is essential. RGB to gray conversion is converted using the function `rgb2gray()`. The initial stage of all image processes is importing the image to the Matlab workspace. Importation of image is done using the function `imread('imagefolderpath')`. 
+
+The following codes are used in importation and image conversion to grayscale. The outcome will be used to demonstrate various edge detection methods.
+
+First, we import the image using the function `imread()` as demonstrated with the below codes.
```

Comment:
Have these explanations after the code. The current way is affecting the general article flow!